s2wav/tiling.py

```diff
@@ -0,0 +1,63 @@
+"""Script defining basic mathematical functions needed to carry out the tiling of the harmonic space. 
```

Comment:
I'm not sure this is standard practice? It is nice to have, but might be worth trying to include this in the function docstrings so it will get pulled into the auto-docs